app/user.go

```diff
@@ -2311,15 +2311,20 @@ func (a *App) GetThreadsForUser(userID, teamID string, options model.GetUserThre
 			return nil
 		})
 
-		eg.Go(func() error {
-			totalCount, err := a.Srv().Store.Thread().GetTotalThreads(userID, teamID, options)
-			if err != nil {
-				return errors.Wrapf(err, "failed to count threads for user id=%s", userID)
-			}
-			result.Total = totalCount
+		// Unread is a legacy flag that caused GetTotalThreads to compute the same value as
+		// GetTotalUnreadThreads. If specified, skip computing total threads and send back
```

Comment:
It's awkward, but I think correct: if `Unread` is not specified, we do this calculation. If specified, we skip it.

That being said, I'll expand for clarity:
```
		// Unread is a legacy flag that caused GetTotalThreads to compute the same value as
		// GetTotalUnreadThreads. If unspecified, do this work normally; otherwise, skip,
		// and send back duplicate values down below.
```